app/controllers/admin/reports_controller.rb

```diff
@@ -61,6 +61,9 @@ def render_data?
     def render_in_background
       cable_ready[ScopedChannel.for_id(params[:uuid])]
         .inner_html(
+          selector: "#report-go",
+          html: helpers.button(t(:go), "report__submit-btn", "submit", disabled: true)
+        ).inner_html(
```

Comment:
I think this could be simpler with a `set-attribute` action (https://cableready.stimulusreflex.com/reference/operations/element-mutations.html#set-attribute)

(If we need to do this with Turbo in the future, turbo_power will have this covered).

Reply:
TIL !